quickstart/csharp/dotnetcore/from-microphone/helloworld/Program.cs

```diff
@@ -3,64 +3,51 @@
 // Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
 //
 
-// <code>
+// <stt-quickstart>
 using System;
+using System.IO;
 using System.Threading.Tasks;
 using Microsoft.CognitiveServices.Speech;
+using Microsoft.CognitiveServices.Speech.Audio;
 
-namespace helloworld
+class Program 
 {
-    class Program
+    static string YourSubscriptionKey = "YourSubscriptionKey";
+    static string YourServiceRegion = "YourServiceRegion";
+
+    async static Task Main(string[] args)
     {
-        public static async Task RecognizeSpeechAsync()
-        {
-            // Creates an instance of a speech config with specified subscription key and service region.
-            // Replace with your own subscription key // and service region (e.g., "westus").
-            var config = SpeechConfig.FromSubscription("YourSubscriptionKey", "YourServiceRegion");
```

Comment:
Please bring back the comments. We did not agree on deleting comments from GitHub quickstarts. If/when we have two versions of quickstarts, one solely dedicated to oneline-docs, we can remove comments from that version. But as long as we only have one version, we need comments in it.